viewer/core/src/utilities/ViewStateHelper.ts

```diff
@@ -14,11 +13,19 @@ import { NodeAppearance } from '@reveal/cad-styling';
 import { CogniteClient } from '@cognite/sdk';
 
 export type ViewerState = {
-  camera: {
+  camera?: {
     position: { x: number; y: number; z: number };
     target: { x: number; y: number; z: number };
   };
-  models: ModelState[];
+  models?: ModelState[];
+  clippingPlanes?: ClippingPlanesState[];
+};
+
+export type ClippingPlanesState = {
+  nx: number;
+  ny: number;
+  nz: number;
```

Comment:
Why we shouldn't use ordinary Vector3?

Reply:
`THREE.Vector3` would add a bunch of unnecessary stuff to the resulting JSON. The point here is to generate a small output as possible, so we use dedicated types.